tests/js/server/recovery/view-arangosearch-link-populate-persistentids-mmfiles.js

```diff
@@ -74,24 +72,17 @@ function recoverySuite () {
 
       var meta = { links: { 'UnitTestsRecoveryDummy': { includeAllFields: true } } };
 
-      let expectFailure = internal.debugCanUseFailAt();
       try {
         view.properties(meta);
-        assertFalse(expectFailure);
+        fail();
       } catch (e) {
-        internal.print(JSON.stringify(e))
-        if (e.errorNum !== arangodb.ERROR_ARANGO_INDEX_CREATION_FAILED){
-          //throw e;
+        if (e.errorNum !== errors.ERROR_ARANGO_INDEX_CREATION_FAILED.code) {
```

Comment:
you could simply use `assertEqual` here. but anyway, the if will also do the job.